tests/linalg_test.py

```diff
@@ -581,6 +581,7 @@ def args_maker():
       for shape in [(1, 1), (4, 4), (2, 70, 7), (2000, 7), (7, 10000), (70, 7, 2)]
       for dtype in float_types + complex_types
       for rng_factory in [jtu.rand_default]))
+  @jtu.skip_on_devices("tpu", "gpu")  # SVD is only implemented on the CPU backend
```

Comment:
SVD is implemented on GPU, too. (So I guess the warning in `linalg.py` is stale.)

Reply:
But the linalg_test_gpu test has already been broken for other reasons (segfault, not this test)